nix_update/__init__.py

```diff
@@ -196,7 +203,13 @@ def validate_git_dir(import_path: str) -> str:
 
 
 def nix_run(options: Options) -> None:
-    cmd = ["nix", "shell", "--extra-experimental-features", "flakes nix-command", "-L"]
+    cmd = [
+        "nix",
+        "shell",
+        "--extra-experimental-features",
+        "flakes nix-command",
+        "-L",
```

Comment:
We should deduplicate the arguments

Reply:
by that do you mean adding `--extra-experimental-features "flakes nix-command"` unconditionally to `system_flags` (maybe rename it to extra_flags)?

Reply:
or just move the list to a common variable and re-use that all over. or what you suggested if we use it everywhere.